classes/04class/exercises/c04-iac01/raghunadhpokkalath/iam.tf

```diff
@@ -0,0 +1,46 @@
+resource "aws_iam_role" "ec2_s3" {
+  name               = "ec2_s3_role"
+  assume_role_policy = <<EOF
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Action": "sts:AssumeRole",
+      "Principal": {
+        "Service": "ec2.amazonaws.com"
+      },
+      "Effect": "Allow"
+    }
+  ]
+}
+EOF
```

Comment:
template files might be useful for bigger policies and other config files https://registry.terraform.io/providers/hashicorp/template/latest/docs/data-sources/file